src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1108,14 +1125,14 @@ private void ProcessOutputRecord(ProcessOutputObject outputValue)
         }
 
         /// <summary>
-        /// Gets the start info for process.
+        /// Gets the ProcessStartInfo for process.
         /// </summary>
-        /// <param name="redirectOutput"></param>
-        /// <param name="redirectError"></param>
-        /// <param name="redirectInput"></param>
-        /// <param name="soloCommand"></param>
-        /// <returns></returns>
-        private ProcessStartInfo GetProcessStartInfo(bool redirectOutput, bool redirectError, bool redirectInput, bool soloCommand)
+        /// <param name="redirectOutput">A boolean which determines whether the output is to be redirected.</param>
+        /// <param name="redirectError">A boolean which determines whether errors are to be redirected.</param>
+        /// <param name="redirectInput">A boolean which determines whether the input is to be redirected.</param>
+        /// <param name="soloCommand">A boolean which determines whether this is a solo command.</param>
+        /// <returns>A ProcessStartInfo object which is the base of the native invocation.</returns>
+        private ProcessStartInfo GetProcessStartInfoObject(bool redirectOutput, bool redirectError, bool redirectInput, bool soloCommand)
```

Comment:
I feel the name of this method was previously just as accurate but more succinct; in OOP, the "Object" suffix is usually tautologous, so doesn't add extra information and is unneeded.

```suggestion
        private ProcessStartInfo GetProcessStartInfo(bool redirectOutput, bool redirectError, bool redirectInput, bool soloCommand)
```

Reply:
I've just seen that this is a new method to be used in the existing `GetProcessInfo()`, but I'm not sure why it's been factored out as a method at this point.

We should work out what the purpose of each method is (by answering the question "what does this method do?" with a single sentence and without the word "and") and name each one in a way that matches that description.